### DISCUSSION TITLE
Saving modified page results in EACCES: permission denied, open 'home.html'

### DISCUSSION THREAD

**Discussion by tobe2bcon** [requarks/wiki#3278] [Error / Bug Report]
Hi there,
I've got wiki.js running with the standard linux installation on a raspberry Pi4 with Debian buster. Everything is working fine so far besides saving a modified page. In this case the above mentioned error occurs. The home.html file is located here: /usr/share/webmin/useradmin/help. The origin user was root. I changed the user to pi (which is my standard user on the raspi) and to wikijs as well. I also changed the user rights for all users, even up to -rwxrwxrwx.

However I am not able to save page modifications. Please advise how to overcome this problem. Thank you very much.

Best
Thomas

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

**Comment by tobe2bcon** [comment]
Sorry, these are the required informations:
OS: Raspbian GNU/Linux 10 (buster)
Wiki.js 2.5.170
MariaDB 10.3.21 (running on Synology DiskStation)
